examples/fmriprep-resample/README.md

```diff
@@ -0,0 +1,108 @@
+
+The example comprises the following files:
+- `fmriprep-resample` template
+- `input.txt` input specification
+- `output.txt` output specification
+- `parameter.txt` parameters
+
+## Requirements
+
+This example requires Singularity.
+
+Please note, that there is no need to install fMRIPrep. The singularity container will be automatically retrieved from the ReproNim containers collection.
+
+## How to install
+
+Install `datalad-remake` extension, as described [here](https://github.com/datalad/datalad-remake/tree/main?tab=readme-ov-file#installation).
+
+## How to use
+
+It is assumed that you have a local copy of the `datalad-remake` project in your `$HOME` directory. If this not the case, adjust the path below:
+
+```
+EXAMPLE=$HOME/datalad-remake/examples/fmriprep-resample
+```
+
+### Clone example dataset
+
+To run the example, you'll need a raw BIDS dataset that has been minimally preprocessed with fMRIPrep. For a complete list of data dependencies, please refer to [this](https://github.com/datalad/datalad-remake/blob/main/examples/fmriprep-resample/input.txt) specification.
+
+For convenience, a ready-made dataset containing all inputs required for running the example can be obtained like so:
+
+```bash
+> cd $HOME
+> datalad clone https://hub.datalad.org/example my-project
+> cd my-project
+> datalad get -n data/ds001734
+> datalad get -n derivatives/ds001734
+```
+
+The dataset is organized in a modular way. It contains raw BIDS data (`data/ds001734`), as well as fMRIPrep derivative data (`derivatives/ds001734`). Also, it includes the `resample.py` script, as well as the software container that will be needed for running the script (`code/containers`).
+
+The resulting dataset structure is as follows:
+
+```
+my-project
+├── code
+│   ├── containers
+│   ├── make
+│   └── resample.py
+├── data
+│   └── ds001734
+└── derivatives
+    └── ds001734
+```
+
+### Configure (or enable) special remote
+
+Configure the dataset in which you want to collect the results of the (re)computation, in this case `derivatives/ds001734` subdataset.
+
+```bash
+> cd $HOME/my-project/derivatives/ds001734
+```
+
+If you're working with the ready-made example dataset, you are good to go. The `datalad-remake` special remote is already enabled. You can verify that by running:
+
+```bash
+> datalad siblings
+```
+
+If you're working with your own dataset, make sure to configure a `datalad-remake` special remote:
+
+```bash
+> git annex initremote datalad-remake type=external externaltype=datalad-remake \
+encryption=none allow-untrusted-execution=true autoenable=true
+```
```

Comment:
remove this section, manual datalad-remake start is not necessary anymore

Reply:
Done in [f45274a](https://github.com/datalad/datalad-remake/pull/62/commits/f45274a254ba9d2c47f9a45ba37df344b6382687).